discovery-provider/src/tasks/index.py

```diff
@@ -348,7 +348,7 @@ def fetch_ipfs_metadata(
             futures_map[future] = [cid, txhash]
 
         try:
-            for future in concurrent.futures.as_completed(futures, timeout=3):
+            for future in concurrent.futures.as_completed(futures):
```

Comment:
Let's remove that timeout error below too

Reply:
will do